src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
 // write your code here
+const bodyElement = document.body;
+
+const population = [...bodyElement.querySelectorAll('.population')];
+const textTotal = bodyElement.querySelector('.total-population');
+const textAverage = bodyElement.querySelector('.average-population');
+
+const value = population.map((elem) => +elem.textContent.replaceAll(',', ''));
+
+const total = value
+  .reduce((sum, elem) => sum + elem, 0)
+  .toLocaleString('en-Us');
```

Comment:
There is an issue with the calculation of the total. The `reduce` method is correctly used to sum up the values, but the result of `reduce` should be converted to a string using `toLocaleString` after the sum is calculated, not during the chaining. Consider storing the result of `reduce` in a variable before converting it to a localized string.